xml/MAIN-SBP-CSP-UpdateInfra.xml

```diff
@@ -483,417 +779,329 @@ ssh-keygen -t rsa -f regionsrv</screen>
+      <step>
+       <para>
+        Copy the mirrored data from the location SMT uses to the location RMT
+        uses:
+       </para>
+      </step>
+      <step>
+<screen>sudo cp -r /var/www/htdocs/repo/* /var/lib/rmt/public/repo</screen>
+      </step>
+     </substeps>
+    </step>
+    <step>
+     <para>
+      Finally, set the permissions on the repository directory so the RMT
+      server can access it:
+     </para>
+<screen>sudo chown -R _rmt:nginx /var/lib/rmt/public/repo</screen>
+    </step>
+   </procedure>
+  </sect2>
+ </sect1>
+ <sect1 xml:id="sec-rmt-registration-sharing-for-csp">
+  <title>Setting up RMT Registration Sharing for Cloud Service Providers</title>
 
-		<sect2 xml:id="sec-intro-rmt-reg-sharing">
-			<title>Introduction</title>
-			<para> Registration sharing replicates client registrations across independent servers.
-				This enables CSPs to provide redundant RMT servers per region so registered
-				instances will continue to have access to valid repositories in case of an RMT
-				failure in the region. </para>
-			<para> For registration sharing you will need to ensure the following: </para>
-			<itemizedlist>
-				<listitem>
-					<para> Two or more RMT servers need to be available in a region that have been
-						deployed using SUSE Linux Enterprise Server 15 SP1.</para>
-				</listitem>
-				<listitem>
-					<para> Each RMT server is registered with SCC. </para>
-				</listitem>
-				<listitem>
-					<para> Each RMT server is synchronizing repositories from SUSE Customer Center
-						and the repositories mirrored must be identical. </para>
-				</listitem>
-				<listitem>
-					<para> Instances can successfully register and deregister with each RMT server
-						in a region. </para>
-				</listitem>
-				<listitem>
-					<para> Instance registrations forwarding to SUSE Customer Center is disabled in
-						each RMT setup. </para>
-				</listitem>
-				<listitem>
-					<!-- TODO: This is not necessary, RMT servers are not necessarily expected to be DNS resolvable
-                                               why do we expect this from CSPs??? -->
-					<para> Name resolution has been configured so all RMT servers in a region can
-						resolve the other RMT servers in the same region. </para>
-				</listitem>
-			</itemizedlist>
-		</sect2>
+  <sect2 xml:id="sec-intro-rmt-reg-sharing">
+   <title>Introduction</title>
+   <para>
+    Registration sharing replicates client registrations across independent
+    servers. This enables CSPs to provide redundant RMT servers per region so
+    registered instances will continue to have access to valid repositories in
+    case of an RMT failure in the region.
+   </para>
+   <para>
+    For registration sharing you will need to ensure the following:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Two or more RMT servers need to be available in a region that have been
+      deployed using SUSE Linux Enterprise Server 15 SP3.
```

Comment:
see earlier comment about version specific reference. This implies you have to update the document for SP4, then SP5 and then SP6.....